packages/rocketchat-livechat/imports/server/rest/sms.js

```diff
@@ -44,6 +44,30 @@ RocketChat.API.v1.addRoute('livechat/sms-incoming/:service', {
 		sendMessage.message.msg = sms.body;
 		sendMessage.guest = visitor;
 
+		if (sms.hasMedia) {
+			sendMessage.message.attachments = [];
+			for (let mediaIndex = 0; mediaIndex < sms.media.length; mediaIndex++) {
```

Comment:
I would suggest to use `sms.media.map()` to populate the `attachments` field to help code reading. but not really a must.